hwilib/devices/trezor.py

```diff
@@ -177,8 +177,13 @@ def sign_message(self, message, keypath):
 
     # Display address of specified type on the device. Only supports single-key based addresses.
     def display_address(self, keypath, p2sh_p2wpkh, bech32):
-        raise NotImplementedError('The HardwareWalletClient base class does not '
-            'implement this method')
+      expanded_path = tools.parse_path(keypath)
+      output = self.client.get_address(
```

Comment:
trezor 0.11.0 moved this to `trezorlib.btc.get_address`.

Reply:
Ok, I'll rebase.